pkg/config/loader_test.go

```diff
@@ -469,6 +473,7 @@ sdsServer:
 				"KUMA_RUNTIME_KUBERNETES_INJECTOR_SIDECAR_CONTAINER_REDIRECT_PORT_INBOUND":                 "2020",
 				"KUMA_RUNTIME_KUBERNETES_INJECTOR_SIDECAR_CONTAINER_REDIRECT_PORT_OUTBOUND":                "1010",
 				"KUMA_RUNTIME_KUBERNETES_INJECTOR_CNI_ENABLED":                                             "true",
+				"KUMA_RUNTIME_KUBERNETES_INJECTOR_SIDECAR_CONTAINER_ENV_VARS":                              "a:b,c:d",
```

Comment:
shouldn't be `;` as a separator here?

Reply:
nope. See how you provide maps to env var https://github.com/kelseyhightower/envconfig

Reply:
👍

Reply:
Is it possible to be consistent here? Can we maybe go with a comma-separated list in the annotations rather than a semicolon?

Reply:
The reason I want to avoid this is:
Imagine DP has a variable `TEST1` and `TEST2`. Both of them are `map[string]string`.
Right now you would be able to set this
```
KUMA_RUNTIME_KUBERNETES_INJECTOR_SIDECAR_CONTAINER_ENV_VARS="TEST1=a:b,b:c;TEST2=a:b,b:c"
```
if we change the separator from `;` to `,`, how would you set such maps?